SQL/Release_patches/23.0_To_24.0_upgrade.sql

```diff
@@ -1,3 +1,8 @@
+SELECT 'Running: SQL/Archive/24.0/2019-07-01_fix_project_in_session.sql';
+
+-- Populate new session field with pre recorded candidate project
+UPDATE session s JOIN candidate c ON s.CandID=c.CandID SET ProjectID=c.RegistrationProjectID WHERE ProjectId IS NULL;
+ALTER TABLE `session` CHANGE `ProjectID` `ProjectID` int(10) unsigned NOT NULL;
```

Comment:
<img width="632" alt="image" src="https://user-images.githubusercontent.com/51176779/154140821-afe8e07d-5996-4261-8310-8be3a53c7bc4.png">

Reply:
<img width="953" alt="image" src="https://user-images.githubusercontent.com/51176779/154141194-813c7a54-9d75-4e8c-aa74-600e0d0641bc.png">

Failing because the scanners also don't have a RegistrationProjectID in candidate table